big_tests/tests/mod_event_pusher_rabbit_SUITE.erl

```diff
@@ -414,20 +414,20 @@ group_chat_message_received_event_properly_formatted(Config) ->
               BobRoomJID = user_room_jid(RoomAddr, Bob),
               AliceJID = client_lower_short_jid(Alice),
               AliceFullJID = client_lower_full_jid(Alice),
-              AliceGroupChatMsgRecvRK = group_chat_msg_recv_rk(AliceJID),
-              Message = <<"Hi there!">>,
-              listen_to_group_chat_msg_recv_events_from_rabbit([AliceJID],
-                                                               Config),
               %% WHEN users chat
```

Comment:
I'd update the comment - here we can add some explanation why users first join the room and only then we subscribe to Rabbit.